src/scenes/Event/store.js

```diff
@@ -97,9 +101,40 @@ export default class EventStore {
 
   @computed get withdrawableAddress() {
     const { currentAddress } = this.app.wallet;
+    const resultIndex = this.event && this.event.currentResultIndex;
+    const profitCut = this.event && this.event.arbitrationRewardPercentage;
+    if (this.betterBets.length === 0 ||
+      this.betterVotes.length === 0 ||
+      this.totalBets.length === 0 ||
+      this.totalVotes.length === 0) return {};
+    const betterBetsSum = sum(this.betterBets);
+    const betterVotesSum = sum(this.betterVotes);
+    const totalBetsSum = sum(this.totalBets);
+    const totalVotesSum = sum(this.totalVotes);
+
+    const totalWinningBets = this.totalBets[resultIndex];
+    const totalLossBets = totalBetsSum - totalWinningBets;
+    const arbitrationShare = (totalLossBets * profitCut) / 100;
+    const betShare = totalLossBets - arbitrationShare;
+    const betterBetsReturn = this.betterBets[resultIndex] + ((betShare * this.betterBets[resultIndex]) / this.totalBets[resultIndex]) || 0;
```

Comment:
when would the `|| 0` ever hit?

Reply:
it would be hit when `this.totalBets[resultIndex]` is 0

Reply:
But this.totalBets[resultIndex] is within the brackets so it shouldn’t be doing any evaluation. Do you mean divide by 0 = infinity and infinity is a falsey value?